### ISSUE TITLE
LLM mode?

### ISSUE THREAD

**Issue by yamatazen** [kijai/ComfyUI-LuminaWrapper#22]
What does the LLM mode do?

**Comment by kijai** [comment]
It's something I was experimenting with, turns out it's unnecessary and we can just use same model for both functions, so I removed the option now.

The idea is that since Gemma is generic LLM model, it can be used for other things than encoding embeds for Lumina. I have this node to experiment with, possible use cases could be prompt improvement etc. 
![image](https://github.com/kijai/ComfyUI-LuminaWrapper/assets/40791699/7749e450-044d-490a-80b9-2b4f57a25696)

**Comment by yamatazen** [comment]
Text generation in ComfyUI?

**Comment by sdk401** [comment]
I was actually going to suggest the same thing, as long as we have the model loaded, is it possible to use it to enhance prompt for Lumina.

> It's something I was experimenting with, turns out it's unnecessary and we can just use same model for both functions, so I removed the option now.

Can you explain, what do you mean "we can just use same model"?
I would like to use that function, but as you removed the node, I can't use "gemma_model" output in anything but "lumina gemma text encode" node. Can I use some other LLM node to load gemma and use it to encode instead of your loader?

**Comment by kijai** [comment]
> I was actually going to suggest the same thing, as long as we have the model loaded, is it possible to use it to enhance prompt for Lumina.
> 
> > It's something I was experimenting with, turns out it's unnecessary and we can just use same model for both functions, so I removed the option now.
> 
> Can you explain, what do you mean "we can just use same model"? I would like to use that function, but as you removed the node, I can't use "gemma_model" output in anything but "lumina gemma text encode" node. Can I use some other LLM node to load gemma and use it to encode instead of your loader?

I mean I initially thought you can't use the "GemmaForCausalLM" -model from transformers for Lumina, which is why I made the switch, but turns out it does work the same as the "GemmaModel" that the transformers "AutoModel" assigns it, which was how it was in the original Lumina code.

And by model I mean the model object in the code, the weights were always the same. 

Anyway all this means just that we can simply use the same loader node for both purposes. Are you saying you don't have the GemmaSampler -node? 
![image](https://github.com/kijai/ComfyUI-LuminaWrapper/assets/40791699/7bcadf4e-78fb-446c-bfdf-4df447ceb8d1)

**Comment by sdk401** [comment]
> Are you saying you don't have the GemmaSampler -node?

Yeah, could not find it. Updated and refreshed anything and now I see it. Thanks!

**Comment by yamatazen** [comment]
Where does the Show Text node come from?

**Comment by kijai** [comment]
> Where does the Show Text node come from?

https://github.com/pythongosssss/ComfyUI-Custom-Scripts
